iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/ShuntCompensatorModification.java

```diff
@@ -56,6 +57,13 @@ public void apply(Network network, boolean throwException, ComputationManager co
         }
     }
 
+    private static void setTargetV(ShuntCompensator shuntCompensator) {
+        if (shuntCompensator.isVoltageRegulatorOn()) {
+            VoltageRegulationUtils.getTargetVForRegulatingElement(shuntCompensator.getNetwork(), shuntCompensator.getRegulatingTerminal().getBusView().getBus(),
+                    shuntCompensator.getId(), IdentifiableType.SHUNT_COMPENSATOR).ifPresent(shuntCompensator::setTargetV);
```

Comment:
Shouldn't we use the voltage of the regulating terminal's bus if `getTargetVForRegulatingElement` returns an empty `Optional`, just like it is done for generators in `Generator.apply()`?

Reply:
I don't think so because if the network cannot provide any interested information, it means that the targetV should come from the network modification of this shunt compensator. And for the moment, the field is not avalable. We did that for generator because it was an existing feature, but not natural for me because it is hidden.

Reply:
Ok, thanks.